packages/api-sdk/.speakeasy/gen.yaml

```diff
@@ -19,7 +19,7 @@ generation:
     oAuth2ClientCredentialsEnabled: true
     oAuth2PasswordEnabled: true
 typescript:
-  version: 0.0.6
+  version: 0.0.0
```

Comment:
note: the version gets auto-bumped based on the `gen.yaml` version. we could either:
1. manually set the version as 0.0.0 whenever we manually build + PR `api-sdk` changes
2. just let the semver auto-bump

while the package used internally and isnt published, we could probably just do (2). once we're ready to publish it, we can reset it to 0.0.0 and then just do (2) since it will naturally follow semver updates.